src/routes/authorRouter.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const { 
+    createAuthor,
+    getAllAuthors,
+    getAuthorById,
+    updateAuthor, 
+    deleteAuthor,
+} = require('../controllers/author');
+
+const authorRouter = express.Router();
+
+authorRouter.route('/')
+.post(createAuthor)
+.get(getAllAuthors);
+
+authorRouter.route('/:id')
+.get(getAuthorById)
+.patch(updateAuthor)
+.delete(deleteAuthor);
```

Comment:
For longer chained routes like this, putting them across multiple lines is definitely preferred but they should each be indented as below:
```suggestion
authorRouter.route('/:id')
  .get(getAuthorById)
  .patch(updateAuthor)
  .delete(deleteAuthor);
```